packages/db/index.js

```diff
@@ -635,10 +637,15 @@ export class DBClient {
       .select(`
         _id:id::text,
         issuer,
-        keys:auth_key_user_id_fkey(_id:id::text, name,secret)
+        pinningEnabled:pinning_enabled,
+        keys:auth_key_user_id_fkey(
+          _id:id::text,
+          name,
+          secret
+        )
       `)
       .match({
-        issuer
+        issuer: issuer
```

Comment:
This shouldn't be needed isn't it? 
If only the value is provided it is implicitly assigned to a property with the same key name.

Reply:
No it isn't needed. Have reverted.